key,default_value
userAgent,"Botocore/1.12.25 Python/2.7.15 Linux/4.16.0-kali2-amd64"
eventID,"2282f47a-8d52-4bc8-a741-230c4ae27ea1"
errorMessage,"Task assessment has not been run or dms-access-for-tasks IAM Role not configured correctly"
userIdentity_type,"IAMUser"
userIdentity_principalId,"AIDADO2GQD0K8TEF7KW1V"
userIdentity_arn,"arn:aws:iam::811596193553:user/Level6"
userIdentity_accountId,"811596193553"
userIdentity_accessKeyId,"AKIA3Z2XBVUDFQ9TU4MD"
userIdentity_userName,"Level6"
eventType,"AwsApiCall"
errorCode,"ResourceNotFoundFault"
sourceIPAddress,"9.245.1.85"
eventName,"DescribeReplicationTaskAssessmentResults"
eventSource,"dms.amazonaws.com"
recipientAccountId,"811596193553"
requestParameters,"None"
awsRegion,"us-east-1"
requestID,"ce0a8a0f-d24a-11e8-b09c-1679d3ecb977"
responseElements,"None"
eventVersion,"1.05"
eventTime,"2018-10-17T20:25:37Z"
